spring-data-cassandra/src/test/java/org/springframework/data/cassandra/test/integration/mapping/BasicCassandraPersistentEntityMetadataVerifierUnitTests.java

```diff
@@ -157,28 +158,24 @@ public boolean equals(Object obj) {
 			return super.equals(obj);
 		}
 
-		@PrimaryKeyColumn(ordinal = 0, type = PrimaryKeyType.PARTITIONED)
-		private String species;
+		@PrimaryKeyColumn(ordinal = 0, type = PrimaryKeyType.PARTITIONED) private String species;
 
-		@PrimaryKeyColumn(ordinal = 1, type = PrimaryKeyType.PARTITIONED)
-		private String breed;
+		@PrimaryKeyColumn(ordinal = 1, type = PrimaryKeyType.PARTITIONED) private String breed;
 
-		@PrimaryKeyColumn(ordinal = 2, type = PrimaryKeyType.CLUSTERED, ordering = Ordering.DESCENDING)
-		private String color;
+		@PrimaryKeyColumn(ordinal = 2, type = PrimaryKeyType.CLUSTERED,
+				ordering = Ordering.DESCENDING) private String color;
```

Comment:
That's the Spring Data formatting. Removing the `private` visibility modifier makes the declaration a one-liner again.